src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.cpp

```diff
@@ -40,6 +38,7 @@
 
 #include <fastdds/dds/log/Log.hpp>
 
+#include <fastdds/builtin/type_lookup_service/TypeLookupManager.hpp>
```

Comment:
This is the only exception. If the header file is the one defining the class that is implemented in the specific source file, then, it should be the first to be included. Please revert the change in this file.

Reply:
Fixed in [7cbeaf0](https://github.com/eProsima/Fast-DDS/pull/4053/commits/7cbeaf000b9780d2985f5b663a0244701a6b5cbe)